cou/logging.py

```diff
@@ -39,15 +40,25 @@ def filter(self, record: logging.LogRecord) -> bool:
         return True
 
 
-def setup_logging(log_level: str = "INFO") -> None:
+def get_log_file() -> Path:
+    """Get log file path.
+
+    :return: Returns log file path
+    :rtype: Path
+    """
+    time_stamp = datetime.now().strftime("%Y%m%d%H%M%S")
+    return Path(f"{COU_DIR_LOG}/cou-{time_stamp}.log")
+
+
+def setup_logging(log_file: Path, log_level: str = "INFO") -> None:
     """Do setup for logging.
 
+    :param log_file: Logging file, defaults to "{COU_DIR_LOG}/cou-{time_stamp}.log"
```

Comment:
I see no default here

```suggestion
    :param log_file: Logging file"
```